sdk/notificationhubs/notification-hubs/src/models/notificationBodyBuilder.ts

```diff
@@ -427,19 +427,29 @@ export interface FirebaseLegacyWebNativePayload {
  * @returns The JSON body to send to Notification Hubs.
  */
 export function createFirebaseLegacyNotificationBody(
-  nativeMessage: FirebaseLegacyNativeMessage,
+  nativeMessage: FirebaseV1NativeMessageEnvelope,
 ): string {
   return JSON.stringify(nativeMessage);
 }
 
+/**
+ * Represents the Firebase V1 native message envelope.
+ */
+export interface FirebaseV1NativeMessageEnvelope {
+  /**
+   * The Firebase V1 native message.
+   */
+  message: FirebaseV1NativeMessage;
+}
+
 /**
  * Represents the targets, options, and payload for HTTP JSON messages for the Firebase V1 interface.
  */
 export interface FirebaseV1NativeMessage {
   /**
    * Custom key-value pairs of the message's payload.
    */
-  data?: Record<string, any>;
+  data?: Record<string, string>;
```

Comment:
weird that the api.md changes don't include these three

Reply:
I forgot to update the API extractor.  Should be fixed now